documentation/insomnia/insomnia-queries.json

```diff
@@ -0,0 +1 @@
ericuzzi\"\n}"},"parameters":[],"headers":[{"name":"Content-Type","value":"application/json","id":"pair_d67c2cb789214bb2a26c5a96d8025a96"}],"authentication":{},"metaSortKey":-1621436358421,"isPrivate":false,"settingStoreCookies":true,"settingSendCookies":true,"settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingRebuildPath":true,"settingFollowRedirects":"global","_type":"request"},{"_id":"req_357086502ec54effa8c2171d1dab436e","parentId":"wrk_caab38ee11b349cabbee914862d5cbd1","modified":1621439747169,"created":1621439737304,"url":"https://ool7sguzj3.execute-api.us-east-1.amazonaws.com/dev/crud","name":"EditUser","description":"","method":"PUT","body":{"mimeType":"application/json","text":"{\n\t\"dob\": \"1983-05-27\",\n\t\"name\": \"Pedro Clericuzzi\"\n}"},"parameters":[],"headers":[{"name":"Content-Type","value":"application/json","id":"pair_d67c2cb789214bb2a26c5a96d8025a96"}],"authentication":{},"metaSortKey":-1621430672505,"isPrivate":false,"settingStoreCookies":true,"settingSendCookies":true,"settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingRebuildPath":true,"settingFollowRedirects":"global","_type":"request"},{"_id":"req_b635841cb93a49c89cb9c9e17b16da77","parentId":"wrk_caab38ee11b349cabbee914862d5cbd1","modified":1621439829226,"created":1621439792837,"url":"https://ool7sguzj3.execute-api.us-east-1.amazonaws.com/dev/crud","name":"Geocoding","description":"","method":"POST","body":{"mimeType":"application/json","text":"{\n\t\"address\": \"Rua Conselheiro Nabuco, 210 Casa Amarela, Recife-PE\",\n\t\"provider\": \"1\"\n}"},"parameters":[],"headers":[{"name":"Content-Type","value":"application/json","id":"pair_d67c2cb789214bb2a26c5a96d8025a96"}],"authentication":{},"metaSortKey":-1621201437539,"isPrivate":false,"settingStoreCookies":true,"settingSendCookies":true,"settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingRebuildPath":true,"settingFollowRedirects":"global","_type":"request"},{"_id":"env_fe08e682d7bd46caa2f51c0f43811015","parentId":"wrk_caab38ee11b349cabbee914862d5cbd1","modified":1620972889946,"created":1620972889946,"name":"Base Environment","data":{},"dataPropertyOrder":null,"color":null,"isPrivate":false,"metaSortKey":1620972171094,"_type":"environment"},{"_id":"jar_d18ba6d9e81948839b0ba6ef7bd3e327","parentId":"wrk_caab38ee11b349cabbee914862d5cbd1","modified":1621166981006,"created":1620972889954,"name":"Default Jar","cookies":[{"key":"ugid","value":"a99e12eb0e22dcb02e28c6ebcff7578a5403887","expires":"2022-05-16T12:09:23.000Z","domain":"unsplash.com","path":"/","secure":true,"extensions":["SameSite=None"],"hostOnly":false,"creation":"2021-05-16T11:59:30.671Z","lastAccessed":"2021-05-16T12:09:41.006Z","id":"7236263086489734"}],"_type":"cookie_jar"},{"_id":"spc_0d2afcf7b715400aa540f8c364fd8ebc","parentId":"wrk_caab38ee11b349cabbee914862d5cbd1","modified":1620972889997,"created":1620972889966,"fileName":"SuperFormula Test","contents":"","contentType":"yaml","_type":"api_spec"}]}
```

Comment:
I don't think that the documentation would be needed, since the documentation already states the calls and what they are for. But, as a learning mechanism for a new member I think it would be good.

I've done it already and with good results, I mean have a new team member document a "not so well documented" api